### DISCUSSION TITLE
How to add a song in the db?

### DISCUSSION THREAD

**Discussion by enchanteddaemon** [Lost-MSth/Arcaea-server#105] [Q&A]
When i add it, it cant download ![Screenshot_2023-04-16-13-41-16-27_97d8557e5edac333d00a63a1df4942ef.jpg](https://user-images.githubusercontent.com/100534753/232273516-c0da259f-f0e1-40cd-8e55-c113cec21c35.jpg)

**Comment by Lost-MSth** [comment]
Well, this is song downloading. Please add related data in `songs` folder.

**Reply by enchanteddaemon** [reply]
i already added but still no work

**Reply by Lost-MSth** [reply]
https://github.com/Lost-MSth/Arcaea-server/wiki/Q&A#why-not-success-in-downloading-songs-or-charts-为什么歌曲或谱面下载失败
